dir.targets

```diff
@@ -10,6 +10,15 @@
   <Target Name="BuildAndTest" DependsOnTargets="Build;Test" />
   <Target Name="RebuildAndTest" DependsOnTargets="Rebuild;Test" />
   <Target Name="Test" />
+  
+  <ItemGroup Condition="'$(IsTestProject)' == 'true'" >
```

Comment:
d'oh!  Will fix right off.  It's meant for it to be != true though... which covers both false and not set.